plugins/modules/vmware_host_scanhba.py

```diff
@@ -37,12 +37,24 @@
     - Cluster name to Rescan the storage subsystem on (this will run the rescan task on each host in the cluster).
     required: false
     type: str
+  rescan_hba:
+    description:
+    - Rescan all host bus adapters for new storage devices. Rescanning all adapters can be slow.
+    required: false
+    default: true
+    type: bool
   refresh_storage:
     description:
     - Refresh the storage system in vCenter/ESXi Web Client for each host found
     required: false
     default: false
     type: bool
+  rescan_vmfs:
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Hi I hope I did this correctly (as this is my first time to contribute). The Fragment is available under https://github.com/ansible/ansible/pull/76368

Reply:
I moved the Fragment into this Pull Request.